src/ralph/models/xapi/constants.py

```diff
@@ -0,0 +1,32 @@
+"""Constants for xAPI specifications"""
+
+# pylint:disable=invalid-name
+
+from typing import Literal
+
+# Languages.
```

Comment:
IMO you can remove dots at the end of such comment.

```suggestion
# Languages
```

Reply:
The same remark applies for all comments of this file.

Reply:
Good point) Thanks) removed the dots)